feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio_nl80211.c

```diff
@@ -14,7 +14,7 @@
 #include <sys/types.h>
 
 #include <linux/sockios.h>
-#include <linux/nl80211.h>
+#include "nl80211-copy.h"
```

Comment:
If we are using same header for WIFI-5 and WIFI-6, what is the point of checking in two headers?